src/index.js

```diff
@@ -1,2 +1,46 @@
 /* eslint-disable react/react-in-jsx-scope, react/jsx-filename-extension */
 /* @jsx createElement */
+
+function createElement(tagName, props, ...children) {
+  const element = document.createElement(tagName);
+
+  Object.entries(props || {}).forEach(([key, value]) => {
+    element[key.toLowerCase()] = value;
+  });
+
+  children.flat().forEach((child) => {
+    if (child instanceof Node) {
+      element.appendChild(child);
+      return;
+    }
+    element.appendChild(document.createTextNode(child));
+  });
+  return element;
+}
+
+function render(count = 0) {
+  const element = (
+    <div id="hello" className="greeting">
+      <p>Hello, world!</p>
+      <p>
+        <button type="button" onClick={() => render(count + 1)}>
```

Comment:
```suggestion
        <button type="button" onClick={handleClick}>
```

만약 함수를 분리해주시면 이렇게 간단하게 표현이 가능할거에요.
그리고 비즈니스 로직이 아무리 복잡해지더라도 유연하게 대응이 가능할거에요.

Reply:
함수의 내용이 없길래 인라인을 하는게 더 괜찮겠지~ 라고 생각했는데 분리하는 쪽이 옳았군요.. !